src/Renderer/Twig/Extension.php

```diff
@@ -285,38 +291,48 @@ public function createUrl($value = null, $options = null): ?string
             }
             $url = $value->getUrl($format, $this->config);
             $url = $base.'/'.ltrim($url, '/');
+
+            return $url;
         } else {
             // string
             if (preg_match('~^(?:f|ht)tps?://~i', $value)) { // external URL
                 $url = $value;
+
+                return $url;
             } else {
-                if (false !== strpos($value, '.')) { // ressource URL (with a dot for extension)
+                if (false !== strpos($value, '.')) { // ressource URL (ie: 'path/style.css')
                     $url = $value;
                     if ($addhash) {
                         $url .= '?'.$hash;
                     }
                     $url = $base.'/'.ltrim($url, '/');
-                } else {
+
+                    return $url;
+                } else { // others cases
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method createUrl uses an else expression. Else is never necessary and you can simplify the code to work without else.](https://app.codacy.com/app/Narno/Cecil/pullRequest?prid=4576548)